lms/djangoapps/certificates/tests/test_tasks.py

```diff
@@ -0,0 +1,21 @@
+from unittest import TestCase
+
+import ddt
+from mock import patch
+
+from lms.djangoapps.certificates.tasks import generate_certificate
+
+
+@ddt.ddt
+class GenerateUserCertificateTest(TestCase):
+    @patch('lms.djangoapps.certificates.tasks.generate_user_certificates')
+    def test_cert_task(self, generate_user_certs_mock):
+        generate_certificate(student='a', course_key='b', otherarg='c', otherotherarg='d')
```

Comment:
Interesting, I didn't see any other task tests like this using `apply_async`. And it fails when I run it... 🤔